include/mbgl/ios/MGLMapboxEvents.h

```diff
@@ -58,4 +54,7 @@ extern NSString *const MGLEventGestureRotateStart;
 //
 + (void) flush;
 
+// Main thread only
++ (void)validate;
+
 @end
```

Comment:
Originally `+pauseMetricsCollection` and `+resumeMetricsCollection` were designed to be called by the developer if necessary. I take it [that’s no longer the case](https://github.com/mapbox/mapbox-gl-native/wiki/Installing-Mapbox-GL-for-iOS/_compare/dd6d6bfd664b9cc3ee7fb32c2ea56854c6a21e2e...7a2a036bc0f66242d294f6972c5fbd932af31207#diff-81e1e4fc5b62518da42b4d3ceb9909faL126)?